{{cookiecutter.project_slug}}/pyproject.toml

```diff
@@ -140,3 +140,146 @@ extend-unsafe-fixes = [
 
 [tool.ruff.lint.isort]
 force-single-line = true
+
+[tool.uv]
+dev-dependencies = [
+  "Werkzeug[watchdog]==3.0.6", # https://github.com/pallets/werkzeug
+  "ipdb==0.13.13", # https://github.com/gotcha/ipdb
+  {%- if cookiecutter.use_docker == 'y' %}
+  "psycopg[c]==3.2.3", # https://github.com/psycopg/psycopg
+  {%- else %}
+  "psycopg[binary]==3.2.3", # https://github.com/psycopg/psycopg
+  {%- endif %}
+  {%- if cookiecutter.use_async == 'y' or cookiecutter.use_celery == 'y' %}
+  "watchfiles==0.24.0", # https://github.com/samuelcolvin/watchfiles
+  {%- endif %}
+  # Testing
+  # ------------------------------------------------------------------------------
+  "mypy==1.11.2", # https://github.com/python/mypy
+  "django-stubs[compatible-mypy]==5.1.0", # https://github.com/typeddjango/django-stubs
+  "pytest==8.3.3", # https://github.com/pytest-dev/pytest
+  "pytest-sugar==1.0.0", # https://github.com/Frozenball/pytest-sugar
+  {%- if cookiecutter.use_drf == "y" %}
+  "djangorestframework-stubs==3.15.1", # https://github.com/typeddjango/djangorestframework-stubs
+  {%- endif %}
+  # Documentation
+  # ------------------------------------------------------------------------------
+  "sphinx==7.4.7", # https://github.com/sphinx-doc/sphinx
+  "sphinx-autobuild==2024.10.3", # https://github.com/GaretJax/sphinx-autobuild
+  # Code quality
+  # ------------------------------------------------------------------------------
+  "ruff==0.6.9", # https://github.com/astral-sh/ruff
+  "coverage==7.6.1", # https://github.com/nedbat/coveragepy
+  "djlint==1.35.2", # https://github.com/Riverside-Healthcare/djLint
+  "pre-commit==3.8.0", # https://github.com/pre-commit/pre-commit
+  # Django
+  # ------------------------------------------------------------------------------
+  "factory-boy==3.3.1", # https://github.com/FactoryBoy/factory_boy
+  "django-debug-toolbar==4.4.6", # https://github.com/jazzband/django-debug-toolbar
+  "django-extensions==3.2.3", # https://github.com/django-extensions/django-extensions
+  "django-coverage-plugin==3.1.0", # https://github.com/nedbat/django_coverage_plugin
+  "pytest-django==4.9.0", # https://github.com/pytest-dev/pytest-django
+]
+
+[project]
+name = "cookiecutter-django"
+version = "2024.10.04"
+description = "A Cookiecutter template for creating production-ready Django projects quickly."
+readme = "README.md"
+
+license = { text = "BSD" }
+authors = [
+  { name = "Daniel Roy Greenfeld", email = "pydanny@gmail.com" },
+]
```

Comment:
That was copied from the template I suppose? These shouk have the variables provided by the user